src/rules/ResourceQuoteStyle.js

```diff
@@ -216,33 +216,39 @@ class ResourceQuoteStyle extends ResourceRule {
             li = new LocaleInfo(locale);
             LICache[locale] = li;
         }
-    
+        /** special case for Japanese */
+        if (locale === "ja-JP") {
+            // accept the main quote style only and also square brackets
+            li.info.delimiter.alternateQuotationStart = "「\\[";
+            li.info.delimiter.alternateQuotationEnd = "」\\]";
```

Comment:
why the `\\`?

Reply:
This string is used in regular expressions, so you have to escape the square brackets.